docs/client/rest/rest_api.md

```diff
@@ -261,6 +261,71 @@ Create an operation with GET_SCHEMAS type
 |:-----------|:----------------------------|:-------|
 | identifier | The identifier of operation | String |
 
+## Operation Resource
+
+### GET /operations/${operationHandle}/event
+
+Get the current event of the operation by the specified operation handle.
+
+#### Response Body
+
+The [KyuubiOperationEvent](#kyuubioperationevent).
+
+### PUT /operations/${operationHandle}
+
+Put an action to the operation
```

Comment:
Perform an action to the pending or running operation. Currently, supported actions are:

- Cancel: to cancel the operation, which means the operation and its corresponding background task will be stopped, and its state will be switched to CANCELED. A canceled operation's status can still be fetched by client requests.
- Close: to close the operation,  which means the operation and its corresponding background task will be stopped, and its state will be switched to CLOSED. A CLOSED operation's status will be removed on the server side and can not be fetched anymore.